domains/rouf0x.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Rouf0x",
+           "email": "gabriel.ruf@gmail.com",
+           "discord": "790712104058617876"
+        },
+    
+        "record": {
+            "CNAME": "https://guns.lol/roufox"
```

Comment:
Invalid CNAME, should be a hostname.

E.g. `dibster.com`

Not: `https://dibster.com/about`